examples/google.rs

```diff
@@ -127,12 +136,33 @@ fn main() {
             );
 
             // Exchange the code with a token.
-            let token = client
+            let token_response = client
                 .exchange_code(code)
                 .set_pkce_verifier(pkce_code_verifier)
                 .request(http_client);
 
-            println!("Google returned the following token:\n{:?}\n", token);
+            println!(
+                "Google returned the following token:\n{:?}\n",
+                token_response
+            );
+
+            // Revoke the obtained token
+            let token_response = token_response.unwrap();
+            let token_to_revoke = if let Some(token) = token_response.refresh_token() {
+                StandardRevocableToken::RefreshToken(token.clone())
```

Comment:
is this the cloning you were asking about? I think it's fine since it's just one token per request, and the alternative would add some lifetime complexity

Reply:
Yes